src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/net45/WebUIFactory.cs

```diff
@@ -34,9 +34,13 @@ internal class WebUIFactory : IWebUIFactory
     {
         public IWebUI CreateAuthenticationDialog(CoreUIParent coreUIParent, RequestContext context)
         {
-            if (coreUIParent.UseHiddenBrowser)
+            if (coreUIParent.CustomWebUi != null )
```

Comment:
General rule when configuring PlatformParamters - if you have set a CustomWebUI, use that. Otherwise, use existing logic to determine which WebUI you want to use.